source/usage-examples/insertMany.txt

```diff
@@ -3,3 +3,57 @@ Insert Multiple Documents
 =========================
 
 .. default-domain:: mongodb
+
+You can insert multiple documents into a collection by using the ``InsertMany()``
+method.
+
+The following example creates a two new document containing a ``title`` and ``text``
+field and inserts it into the ``haikus`` collection of the ``insertDB`` database:
+
+.. note:: Non-existent Database and Collections
+
+    The driver automatically creates the necessary database and/or collection
+    when you perform a write operation against them if they don't already exist.
+
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/insertMany.go
+   :start-after: begin insertMany
+   :end-before: end insertMany
+   :emphasize-lines: 7
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/insertMany.go>`__ to see a fully runnable example.
```

Comment:
To check, is this link correct? Clicking on the staging page receives a 404.

Reply:
It is. These links won't work until the PR is merged and everything is in the master or version branch since it assumes that location.